extended/src/main/java/apoc/get/GetProcedures.java

```diff
@@ -20,13 +21,13 @@ public class GetProcedures {
     @Procedure
     @Description("apoc.get.nodes(node|id|[ids]) - quickly returns all nodes with these id's")
     public Stream<NodeResult> nodes(@Name("nodes") Object ids) {
-        return new Get(tx).nodes(ids);
+        return new Get((InternalTransaction) tx).nodes(ids);
```

Comment:
is this related or unrelated change?

Reply:
Yes sorry, it is not related to the PR indeed, it's needed to fix a compilation error due to a 5.13 core change,
I'll put it in a separate PR along with the other errors I'm fixing